stitch_assets/code/Login_Screen.html

```
@@ -0,0 +1,219 @@
+<style>
+    body {
+      min-height: max(884px, 100dvh);
+    }
+  </style>
+  </head>
+<body class="bg-background font-body text-on-background min-h-screen flex flex-col md:flex-row overflow-x-hidden">
+<!-- Suppressed Shell: BottomNavBar and TopAppBar are excluded for login journey per Rule -->
+<!-- Left Column: Visual/Marketing (Hidden on Mobile) -->
+<section class="hidden md:flex md:w-1/2 relative overflow-hidden bg-primary-container items-center justify-center p-12">
+<div class="absolute inset-0 z-0">
+<img class="w-full h-full object-cover opacity-30 mix-blend-overlay" data-alt="Modern collaborative workspace with diverse professionals exchanging ideas over digital tablets in a bright airy studio setting" src="https://lh3.googleusercontent.com/aida-public/AB6AXuB5wkrUP8Zafh_g-FIoQzlufiIbG_3b9c1cCUIyeJ7B4YSA_5H4MsmfrmQ2C2hmoswwCIvjHhMUbTg5hnIdNMTG5f6cKH4vTxfUhLKpNRFWcTvlAEtzTCFrNCC2uyxUSKrnMnurxDKQAkfqjT-SPR3NK41Hp1HsCJZs8Oyd0_Y9EaSuNhFTJreRa_d-a7KHrPqObFjoHV7pxVfJrOU1AeYTWs9BfO09JlQrcB-NbsMC1nb_BMo6F11mC_I0OycsCWDTmz330CFfUDY"/>
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)